src/library_pthread.js

```diff
@@ -402,8 +402,18 @@ var LibraryPThread = {
 #if PTHREADS_DEBUG
         out('Allocating a new web worker from ' + new URL('{{{ PTHREAD_WORKER_FILE }}}', import.meta.url));
 #endif
-        // Use bundler-friendly `new Worker(new URL(..., import.meta.url))` pattern; works in browsers too.
+#if TRUSTED_TYPES
+        // Use Trusted Types compatible wrappers.
+        if (typeof trustedTypes !== 'undefined' && trustedTypes.createPolicy) {
+          var p = trustedTypes.createPolicy('emscripten#workerPolicy1', { createScriptURL: function(ignored) { return new URL('{{{ PTHREAD_WORKER_FILE }}}', import.meta.url)} });
+          // Use bundler-friendly `new Worker(new URL(..., import.meta.url))` pattern; works in browsers too.
```

Comment:
I don't understand this comment - there is no `new URL(` here?

Reply:
You're right, it's a copy-paste artifact. Removing!

Reply:
Thanks so much!